src/entity/Feature.ts

```diff
@@ -6,6 +6,11 @@ export enum FeatureType {
   Search = 'search',
 }
 
+export enum FeatureValue {
+  Up = 1,
+  Down = -1,
```

Comment:
Better names I guess
```suggestion
  Allow = 1,
  Block = -1,
```